src/components/HTMLEngineProvider/HTMLRenderers/ImageRenderer.tsx

```diff
@@ -113,7 +113,16 @@ function ImageRenderer({tnode}: ImageRendererProps) {
                                 if (isDisabled) {
                                     return;
                                 }
-                                showContextMenuForReport(event, anchor, report?.reportID, action, checkIfContextMenuActive, ReportUtils.isArchivedRoom(report, reportNameValuePairs));
+                                return onShowContextMenu(() =>
+                                    showContextMenuForReport(
+                                        event,
+                                        anchor,
+                                        report?.reportID ?? '-1',
```

Comment:
```suggestion
                                        report?.reportID ?? CONST.DEFAULT_NUMBER_ID,
```

Reply:
check the new commit

Reply:
One suggestion for you that you need to have meaningful commit message instead repeating `fix cors errors on attachments`.

Reply:
That's really confusing